content/conditions/type-2-diabetes/finding-help-and-support/main-content.md

```diff
@@ -0,0 +1,55 @@
+## Finding help and support for type 2 diabetes
+
+There is a lot of information and support available for type 2 diabetes.
+Some of the support depends on the area you live in.
+
+### Take a course to help you manage your diabetes
+
+There are free education courses to help you learn more about and manage your
+type 2 diabetes.
+
+Your GP will need to refer you. You can phone your GP surgery to get a referral
+letter, you don’t need to make an appointment.
+
+Read more information about [education courses for type 2 diabetes](http://www.desmond-project.org.uk/newlydiagnosedandfoundationmodules-278.html)
+
+### Telling DVLA you have type 2 diabetes
+
+If you're taking insulin for your type 2 diabetes, you will need to
+[tell the DVLA](https://www.gov.uk/diabetes-driving). This is because of the
+risk of hypoglycaemia. You can be fined if you don't tell DVLA.
+
+### Support groups for type 2 diabetes
+
+The charity Diabetes UK runs [local support groups.](https://www.diabetes.org.uk/How_we_help/Local_support_groups/)
+
+They can help with things like managing your diabetes on a daily basis, diet,
+exercise or dealing with emotional problems like depression. They’re a place to
+talk and find out how others live with the condition.
+
+### Blogs and forums about diabetes
+
+* [Diabetes.co.uk forum](http://www.diabetes.co.uk/forum/) - discussions about
+  living with and managing diabetes
+* [Diabetes UK blogs](http://blogs.diabetes.org.uk/) - collection of different
+  blogs on food, eyes, work and diabetes and more
+* [Diabetes Chat](http://www.diabetes.co.uk/diabetes-chat/) - offers scheduled
+  chats with healthcare professionals or just the chance to talk to others
+
+### Telling others can be difficult
+
+However, it can help for certain people to know you have diabetes:
+
+* family can support you, especially as you will need to make change what you eat
```

Comment:
"...you will need to make [a] change to what you eat" ?

"...you will need to make change[s] to what you eat" ?